source/manual/aws-console-access.html.md

```diff
@@ -0,0 +1,57 @@
+---
+owner_slack: "#govuk-2ndline"
+title: Access the AWS console
+section: AWS accounts
+layout: manual_layout
+parent: "/manual.html"
+last_reviewed_on: 2018-10-10
+review_in: 3 months
+redirects:
+  - /manual/seeing-things-in-the-aws-console.html
+---
+
+💡 Before you can access a thing in AWS, you [need to set up your AWS account](/manual/set-up-aws-account.html).
```

Comment:
Not sure on what a good alternative could be, but `Before you can access a thing in AWS` doesn't sound too well. Maybe `Before you can access a service in AWS`, or  `Before you can access something in AWS`?

Reply:
Done, thanks!